client/executor/wasmtime/src/tests.rs

```diff
@@ -171,3 +178,143 @@ fn test_stack_depth_reaching() {
 		format!("{:?}", err).starts_with("Other(\"Wasm execution trapped: wasm trap: unreachable")
 	);
 }
+
+#[test]
+fn test_max_memory_pages() {
+	fn try_instantiate(
+		wat: &'static str,
+		max_memory_pages: Option<u32>,
```

Comment:
nit: I would swap those arguments to have the long one last.

Reply:
Ah, thanks for that! This thought also crossed my mind but it wasn't enough for me to make the change.